exercises/bowling/example.tt

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'bowling'
+
+# Test data version:
+# <%= sha1 %>
+class BowlingTest < Minitest::Test
+  def setup
+    @game = Game.new
+  end
+
+  def roll(rolls)
+    rolls.each { |pins| @game.roll(pins) }
+  end
+<% test_cases.each do |test_case| %>
+  def <%= test_case.name %><% if test_case.skipped? %>
```

Comment:
Also we're trying to standardise the method names in generators, so using `test_name`, `skipped`, `work_load` is preferred.